lib/index.js

```diff
@@ -437,7 +437,7 @@ module.exports.renderSync = function(opts) {
     return result;
   }
 
-  throw util._extend(new Error(), JSON.parse(result.error));
+  throw assign(new Error(), JSON.parse(result.error));
```

Comment:
Yup, the reason we are returning string is because libsass returns us string (via `sass_context_get_error_json`) and we would need to parse the JSON in C++ in v8 binding.

Now thinking about it more, there are two approaches we can avoid this new Error()+ loadash.assign() overhead in JS code:

1. use `sass_context_get_error_[message/line/column]` functions and construct the `Error` typed object in C++. I used those in LibSass.NET and wrote some basic tests around them and they works (i.e. no latent bugs lurking in those API functions).
2. keep using `sass_context_get_error_json` from LibSass and use `v8::JSON::Parse` (which I happen to just found out) to create an `Error` typed object in C++.

I would go with option 1 as LibSass already have those constituents available in memory and probably even lesser overhead constructing an `Error` object and assign values than constructing and parsing the values in C++.

This is me just being super pedantic, JSON.parse in JS is sufficient and jsonParse+lodashAssign is probably not incurring a significant overhead; but doing it in C++ side would render errors would be bit faster in stress testing but not in real world scenarios. If you guys think we should greedily save some cycles and generally care for such [in]significant optimization oppertunities, I can make a patch over the weekend on top of it. Otherwise, this SGTM. 😸